docs/overview.md

```diff
@@ -1,80 +1,78 @@
-
-These pieces are deeply interlinked. Fortunately, we can express the web of relations visually. The first four components - Schema, Sequence, Lens, and Lineage - are easy to think about independent of any particular program trying to do some useful operation. If we had a valid Lineage containing two Sequences, each containing two Schema, and thus necessarily one explicitly-defined Lens, we could visualize it as:
+These logical components are deeply interlinked as illustrated below. The first four components - schema, sequence, lens, and lineage - can be understood independent of a program that is running a useful operation. If we had a valid lineage containing two sequences, each containing two schema, and thus necessarily one explicitly-defined lens, it would look like this:
 
 ![Abstract Lineage](lineage-structure.png)
 
-Lenses within a Sequence are implicit, because that's precisely what backwards compatibility means: Instances of old Schema are still valid with respect to newer Schema, without the need for translation. (NOTE: diagram notwithstanding, this is clearly not true in reverse - this is a [TODO](https://github.com/grafana/thema/issues/6).)
+Lenses within a sequence are implicit, because that's precisely what backward compatibility means: instances of old schema remain valid with respect to newer schema, without the need for translation. (NOTE: diagram notwithstanding, this is clearly not true in reverse - this is a [TODO](https://github.com/grafana/thema/issues/6).)
 
-The remaining two concepts - Instance and Lacuna - are best illustrated in the context of a program performing operations on a Lineage. Beyond that, it's best to look at concrete examples, either via the [Lineage authoring tutorial](authoring.md) or by looking at the [exemplars](https://github.com/grafana/thema/tree/main/exemplars).
+The remaining two concepts - instance and lacuna - are most easily understood in the context of a program performing operations on a lineage. Beyond that, it's useful to review concrete examples, either via the [Lineage authoring tutorial](authoring.md) or by looking at the [exemplars](https://github.com/grafana/thema/tree/main/exemplars).
 
-## Operations
+## About Thema Operations
 
-Thema has three essential operations, given a valid Lineage:
+Thema has three essential operations, given a valid lineage:
 
-* **Pick():** get a particular Schema from a Lineage by version number.
-* **Validate():** check whether some data is an Instance - whether it's valid with respect to a particular Schema.
-* **Translate():** given an Instance of some Schema in a Lineage, transform it to an Instance of some other Schema in the same Lineage, and emit any Lacuna arising from the transformation.
+* **Pick():** get a particular schema from a lineage by version number.
+* **Validate():** check whether some data is an instance to identify whether it's valid with respect to a particular schema.
```

Comment:
Do we validate against a schema or a lineage?

Reply:
Schema, same as in any other system.

It's a reasonable question, given the current framing. i think the other diagram will help (i have to rewrite my graffle a bit so that it really works as a GIF).

So, while data is validated to be an instance by checking against a particular schema, it's _also_ possible to [`SearchAndValidate()`](https://pkg.go.dev/github.com/grafana/thema#section-readme), where you hand some possibly-valid data to an entire lineage, which will traverse the lineage attempting to validate against all the contained schemas. 

To me, this is one of the helpers "composed from these three parts", below. But that's clearly obvious only to me, and probably deserves mention at this high level.

Reply:
will also help with this "three step process" explanation